tests/schemas/test_note_schema.py

```diff
@@ -0,0 +1,39 @@
+import pytest
+from schemas.notes import *
+from models.notes import NotesModel
+
+
+@pytest.mark.usefixtures("empty_test_db")
+class TestNotesSchemaSerialization:
```

Comment:
Thanks for adding tests! These tests look good. I'm not sure if the deserialization test is all that useful since it is testing default marshmallow behavior, however I don't see any harm in leaving it in for now.

Reply:
I initially included the deserialization test because I was over writing the default behavior for 'user'. But I've since change 'user' to 'userid', so this test isn't necessary any more.